src/Consolonia.Gallery/View/ControlsListView.axaml

```diff
@@ -4,6 +4,7 @@
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         mc:Ignorable="d"
+        WindowStartupLocation="CenterScreen"
```

Comment:
So what does it do eventually, what is the effect?

Reply:
it doesn't blow up. 

As I told you in other PR, I've been experimenting with adding .Console platform projects to existing avalonia projects.  (A UI is shared between Desktop, Console, Android, etc)

WindowStartupLocation is a hint to the platform. On Desktop it positions the window. It's ignored by Android, and ignored by Console app.

Reply:
Ah, understood, then add //todo: raise ConsoloniaNotSupportedException
please.

Reply:
Where would I leave it? It's Avalonia that throws because there the environment isn't set up correctly.  It expects there to be Screens defined by the platform so it can do screen based calculations. 
This PR implements the missing platform Screens functionality. 

The other place is that it generates a request to Resize() and Move() on the window. 
The current code for Resize looks like this:
```csharp 
  public void Resize(Size clientSize, WindowResizeReason reason = WindowResizeReason.Application)
  {
      // todo: can we deny resizing? TODO We could consider resizing the console window
  }
```
and I changed Move to be same
```csharp
  public void Move(PixelPoint point)
  {
      // ignore move request, TODO We could consider moving the console window
  }
```